.github/workflows/yehezgun.yml

```diff
@@ -0,0 +1,38 @@
+name: Template Git Actions
+
+run-name:
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  Base-Actions:
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "🎉 The job was automatically triggered by a ${{ github.event_name }} event"
+
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
```

Comment:
Should we use Node.js 18 instead of Node.js 16 for this?

Reply:
Updated to Node 18